src/components/input/bl-input.ts

```diff
@@ -84,13 +84,13 @@ export default class BlInput extends FormControlMixin(LitElement) {
   /**
    * Sets the smallest number can be entered to a `number` input
    */
-  @property({ type: Number, reflect: true })
+  @property({ reflect: true })
   min?: number;
```

Comment:
Don't we need to set a union type as `number | string` to allow date style input for this?

Reply:
I've additionally added 'date' to type now.